terraform/eks/daemon/app_signals/variables.tf

```diff
@@ -34,4 +34,9 @@ variable "ami_type" {
 variable "instance_type" {
   type    = string
   default = "t3a.medium"
-}
\ No newline at end of file
+}
+
+variable "hop_limit" {
```

Comment:
All these are here because we pass in hop limit through the gha and w/o this the tests will not execute